static/js/reload.js

```diff
@@ -29,9 +29,6 @@ function preserve_state(send_after_reload, save_pointer, save_narrow, save_compo
         return;
     }
 
-    if (send_after_reload === undefined) {
-        send_after_reload = 0;
-    }
     let url = "#reload:send_after_reload=" + Number(send_after_reload);
```

Comment:
I guess `Number(false) = 0`; so this doesn't have a behavior change.  We perhaps should think about changing this weird encoding logic more broadly, but this is a noop.